.travis.yml

```diff
@@ -1,15 +1,8 @@
 sudo: false
 language: node_js
 node_js:
-  - stable
-  - 7.6
-matrix:
-  include:
-    - node_js: "8.0"
-      env: "NVM_NODEJS_ORG_MIRROR=https://nodejs.org/download/nightly"
-  allow_failures:
-    # Allow the nightly installs to fail
-    - env: "NVM_NODEJS_ORG_MIRROR=https://nodejs.org/download/nightly"
```

Comment:
There's no point in testing and actively developing against node 6 if 7.6> is the target. There's however nothing working against a user to themselves transpile Koa 2 down to node 6.